datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/eula.py

```diff
@@ -25,7 +26,7 @@ def eula(check):
     echo_info(f"Validating EULA files for {len(checks)} checks...")
 
     for check_name in checks:
-        eula_relative_location, eula_exists = get_eula_from_manifest(check_name)
+        eula_relative_location, eula_exists = ManifestGateway.load_manifest(check_name).get_eula_from_manifest()
```

Comment:
In this case it wasn't needed explicitly because EULA validations were only run for marketplace. But I added it for consistency and it doesn't hurt to keep the extra validation